src/AspNet.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Discovery.cs

```diff
@@ -246,6 +246,13 @@ public partial class OpenIdConnectServerHandler
 
             notification.Scopes.Add(OpenIdConnectConstants.Scopes.OpenId);
 
+            notification.Claims.Add(OpenIdConnectConstants.Claims.Audience);
```

Comment:
@NinoFloris let me know if you can think of any other claim.

Reply:
iss, iat, exp? and maybe but really maybe sub?

Reply:
Already included. Expand the lines to see the complete list :sweat_smile:

Reply:
yep (edited comment) but in that case LGTM